programmers-coding-test/kimjiwon/알고리즘/거스름돈.js

```diff
@@ -0,0 +1,32 @@
+function giveChange(customerMoney, goodsPrice) {
+    const unitMoneys = [10000 ,5000 ,1000, 500, 100, 50, 10];
+    const changeUnitMoney = new Map();
+
+    let sum = 0,
+    change = customerMoney - goodsPrice;
+
+    unitMoneys.forEach(unitMoney => {
+        if (calcurate(change, unitMoney)) {
+            changeUnitMoney.set(`${unitMoney}`, calcurate(change, unitMoney));
+            change -= (unitMoney * calcurate(change, unitMoney)); 
+        }
+    });
```

Comment:
calculate라는 함수가 없으므로 밑에서 선언한 countSpendingMoneys 함수로 대체

```suggestion
    unitMoneys.forEach(unitMoney => {
        if (countSpendingMoneys(change, unitMoney)) {
            changeUnitMoney.set(`${unitMoney}`, countSpendingMoneys(change, unitMoney));
            change -= (unitMoney * countSpendingMoneys(change, unitMoney)); 
        }
    });
```